src/QMCHamiltonians/OperatorBase.h

```diff
@@ -360,7 +360,7 @@ class OperatorBase : public QMCTraits
    * @return Void
    */
   inline virtual void evaluateOneBodyOpMatrixForceDeriv(ParticleSet& P,
-                                                        const ParticleSet& source,
+                                                        ParticleSet& source,
```

Comment:
Is this because of the jastrow implementation need to move the ions and thus the const has to be dropped.
Please create an issue to document the issue of current Jastrow implementation. Please mention the routine name and which Jastrows are affected.

Reply:
I think that's right:  3-body jastrows compute ion derivatives from finite differences at the moment.  I'll make an issue.

Reply:
Put up  #3928